en/c-team-codex.md

```diff
@@ -0,0 +1,53 @@
+# C-Team Codex
+
+The C-Team in Applifting is a group of dedicated leaders who guide the company's direction and embody our core values. They ensure Applifting runs smoothly, grows sustainably, and stays true to its teal principles. By living the ideals in this codex, the C-Team drives Applifting towards its mission of creating digital products with positive impact on planetary scale.
+
+## Lead with Vision, Not Control
+
+### Trust in Teal culture
+
+Lead by setting a clear direction, not by micromanaging. Applifters should be empowered to make decisions and take responsibility.
+
+Your role is to inspire and to remove obstacles. Instead of managing people, mentor them. Help others find their path to success, be it through advice, knowledge, or resources.
+
+Value each person's contribution, regardless of title or status. Every voice in the organization is important.
```

Comment:
```suggestion
Value each person's contribution, regardless of title or status. Every voice in the organization should be heard.
```
Because the importance of each contribution is assessed individually and if that voice/opinion is against other values, than it is not important and would slow all down.